tests/test_virtuaaliviivakoodi.py

```diff
@@ -69,6 +69,168 @@ class TestVirtuaaliviivakoodi(TestCase):
                 },
                 "549500094200287300000000007000000000999999999993201212",
             ),
+            (
```

Comment:
There seems to be something off with the formatting -> install pre-commit hooks from the repo root and format your changes. I've been too lazy to setup any Github actions to this repo 😄

Reply:
Yep, got too used to the automatic formatting :D